Simplifying dockerfile, minor bug fixes

markremmey opened the pull request:
- Removed certain default tool configurations that were resulting in excessively long docker builds
- Corrected a typo regarding "apiversion" in LLM call
- Added .dockerignore

placerda commented:
Hi @markremmey thanks for the updates! Before we merge, could you please address a few small adjustments to keep consistency across repositories?

Update the README.md — the current content is a bit confusing (it still mentions Chris’s fork and Release 1 components such as the Function App). You can use the orchestrator’s README as a reference:
[gpt-rag-orchestrator/README.md](https://github.com/Azure/gpt-rag-orchestrator/blob/main/README.md)

Update the CHANGELOG.md — please add a new entry for version 0.2.3 with the changes included in this PR.
[gpt-rag-mcp/CHANGELOG.md](https://github.com/Azure/gpt-rag-mcp/blob/main/CHANGELOG.md)

Add a VERSION file in the project root with the value 0.2.3, following the orchestrator’s structure:
[gpt-rag-orchestrator/VERSION](https://github.com/Azure/gpt-rag-orchestrator/blob/main/VERSION)

Thanks again for the contribution!

markremmey commented:
Hello @placerda 

I have updated CHANGELOG, README.md, and found some version issues that I've corrected in requirements.txt.

Will plan to make more updates to docs in coming weeks.
